src/main/java/org/broadinstitute/hellbender/engine/filters/ReadGroupBlackListReadFilter.java

```diff
@@ -48,27 +50,22 @@
     public ReadGroupBlackListReadFilter(final List<String> blackLists, final SAMFileHeader header) {
         super.setHeader(header);
         this.blackList.addAll(blackLists);
-        final Map<String, Collection<String>> filters = new TreeMap<>();
-        for (String blackList : blackLists) {
-            try {
-                addFilter(filters, blackList, null, 0);
-            } catch (IOException e) {
-                throw new UserException("Incorrect blacklist:" + blackList, e);
-            }
-        }
-        //merge all the new entries in to the blacklist
-        filters.forEach((k, v) -> blacklistEntries.merge(k, v, (v1, v2) -> {v1.addAll(v2); return v1;}));
+        parseReadGroupFilters();
     }
 
-    private void addFilter(final Map<String, Collection<String>> filters, final String filter, final File parentFile, final int parentLineNum) throws IOException {
-        if (filter.toLowerCase().endsWith(".list") || filter.toLowerCase().endsWith(".txt")) {
-            addFiltersFromFile(filters, filter);
-        } else {
-            addFiltersFromString(filters, filter, parentFile, parentLineNum);
+    private void parseReadGroupFilters() {
+        final Map<String, Collection<String>> filters = new TreeMap<>();
+        for (String blackList : this.blackList) {
+                addFilters(filters, blackList, null, 0);
         }
+        //merge all the new entries in to the blacklist
+        filters.forEach((k, v) -> blacklistEntries.merge(k, v, (v1, v2) -> {
+            v1.addAll(v2);
+            return v1;
+        }));
     }
 
-    private void addFiltersFromString(final Map<String, Collection<String>> filters, final String filter, final File parentFile, final int parentLineNum) {
+    private void addFilters(final Map<String, Collection<String>> filters, final String filter, final File parentFile, final int parentLineNum) {
```

Comment:
parentFile and parentLineNum aren't actually used and should be removed

Reply:
Done.